quiz/envs/backend.default.env

```diff
@@ -1,3 +1,9 @@
 DJ_SECRET_KEY=
```

Comment:
Instrukcje jak wypełnić te envy / defaulty tam gdzie to mozliwe
Bo ludziom apki nie wstają